virttest/utils_net.py

```diff
@@ -518,6 +568,29 @@ def set_netmask(self, netmask):
                             socket.AF_INET, b'\x00' * 2, nmbytes, b'\x00' * 8)
         fcntl.ioctl(sockfd, arch.SIOCSIFNETMASK, ifreq)
 
+    @warp_init_del
+    def get_mtu(self):
+        '''
+        Get MTU size of the interface
+        '''
+        ifreq = struct.pack('16sH14s', self.name.encode(),
+                            socket.AF_INET, b'\x00' * 14)
+        try:
+            res = fcntl.ioctl(sockfd, arch.SIOCGIFMTU, ifreq)
+        except IOError:
+            return None
+
+        return struct.unpack('16sH14s', res)[1]
+
+    @warp_init_del
+    def set_mtu(self, newmtu):
```

Comment:
In common cases, if we want to set mtu in guest, we should set mtu on host tap0 either.
It will be like:
interface.set_mtu()
process.system_output("ifconfig tap0 mtu <value>")

Is that how you use the function? Please enlighten me with the usage, thanks.

Reply:
@vivianQizhu @xiywang Thanks for your review.
If we want to set the mtu size of tap0, it will be like `Interface(tap0).set_mtu(1500)`
And I will update the codes according to your suggestions above.